src/_includes/font-page.njk

```diff
@@ -3,21 +3,21 @@ layout: base
 ---
 
 {%- for theme in samples.themes -%}
-  <input type="radio" 
```

Comment:
I like the upgrade here! But can we clean up the noise in the changes here?

Reply:
yes, no problem.

just to confirm you are talking about adding back the spaces at the end of some of the lines in this file? The cleanup i was doing was removing those extra spaces after line endings, etc. See below, that's all that is being removed/cleanedup.

<img width="452" alt="Screen Shot 2020-12-04 at 10 28 59 AM" src="https://user-images.githubusercontent.com/1774589/101181998-8d71e280-361b-11eb-99ee-52f16b4acab0.png">